package.json

```diff
@@ -1,5 +1,12 @@
 {
-    "dependencies": {
-        "yuglify": "^0.1.4"
-    }
+  "scripts": {
+    "test": "eslint dashboard/static/js/*.js; csslint --quiet --warnings=' ' dashboard/static/css/*.css push/static/css/*.css"
```

Comment:
We'll have to be careful to keep these file paths up-to-date as we add more CSS and JS.